instances/latest/terminologies/disease/macularDegeneration.jsonld

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.om-i.org/props/"
+  },
+  "@id": "https://openminds.om-i.org/instances/disease/macularDegeneration",
+  "@type": "https://openminds.om-i.org/types/Disease",
+  "definition": "Degenerative changes in the macula lutea of the retina.",
+  "description": "Deterioration of the macula lutea in the retina; may be inherited, drug induced, or due to aging; leads to a severe loss of central vision while peripheral vision is retained.",
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0106448",
+  "knowledgeSpaceLink": null,
+  "name": "Macular degeneration",
+  "preferredOntologyIdentifier": "http://evs.nci.nih.gov/ftp1/NDF-RT/NDF-RT.owl#N0000001918",
+  "synonym": [
+"Age-Related Macular Degeneration",
+"Age-Related Maculopathies",
+"Age-Related Maculopathy",
+"Macular Dystrophy",
+"Maculopathies, Age-Related",
+"Maculopathy, Age-Related"
+  ]
```

Comment:
- not capitalized 
- The age-related synonyms have to be removed. Your description gives alternative causes, so the age-related one is only a subtype of this term (also confirmed by the disease ontology: 'macular degeneration' --> 'degeneration of macula and posterior pole --> 'age related macular degeneration' with 15 specific entries).  
  - keep: maculopathy, macular dystrophy
  - add: macular degeneration of retina (from DOID)
  - (sort them alphabetically as you already did for the previous list)